iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -91,6 +111,66 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
+            // For the purpose of this sample, we'll only check the writable property versions between local and server
+            // side without comparing the property values.
+            if (serverWritableProperties > s_localWritableProperties)
+            {
+                _logger.LogDebug($"The writable property version cached on local is changing from {s_localWritableProperties} to {serverWritableProperties}.");
+
+                foreach (KeyValuePair<string, object> writableProperty in writableProperties)
+                {
+                    switch (writableProperty.Key)
+                    {
+                        case Thermostat1:
+                        case Thermostat2:
+                            const string targetTemperatureProperty = "targetTemperature";
+                            string componentName = writableProperty.Key;
+
+                            if (writableProperties.TryGetValue(componentName, targetTemperatureProperty, out double targetTemperatureValue))
+                            { 
+                                _logger.LogDebug($"Property: Received - component=\"{componentName}\", [ \"{targetTemperatureProperty}\": {targetTemperatureValue}°C ].");
+
+                                _temperature[componentName] = targetTemperatureValue;
+
+                                string propertyValueString = $"{{ \"value\": {_temperature[componentName]}, \"ac\": {CommonClientResponseCodes.OK}, \"av\": {serverWritableProperties} }}";
+                                var propertyValue = SystemTextJsonPayloadSerializer.Instance.DeserializeToType<object>(propertyValueString);
+
+                                var reportedProperty = new ClientPropertyCollection();
+                                reportedProperty.AddComponentProperty(componentName, targetTemperatureProperty, propertyValue);
+
+                                ClientPropertiesUpdateResponse updateResponse = await _deviceClient.UpdateClientPropertiesAsync(reportedProperty);
+
+                                _logger.LogDebug($"Property: Update - component=\"{componentName}\", {reportedProperty.GetSerializedString()} is {nameof(CommonClientResponseCodes.OK)} " +
+                                    $"with a version of {updateResponse.Version}.");
+
+                                break;
+                            }
+                            else
+                            {
+                                _logger.LogWarning($"Property: Received an unrecognized property update from service for component {componentName}:" +
+                                            $"\n[ {writableProperty.Value} ].");
+                                break;
+                            }
+
+                        default:
+                            _logger.LogWarning($"Property: Received an unrecognized property update from service:" +
+                                            $"\n[ {writableProperty.Key}: {writableProperty.Value} ].");

```

Comment:
seems like a lot of empty spaces before the second line.

Reply:
standard would be just 4 (one indent from parent line)

Reply:
Corrected for other multiple-empty-spaces logging format as well.